src/main/scala/com/wavesplatform/account/KeyPair.scala

```diff
@@ -0,0 +1,36 @@
+package com.wavesplatform.account
+
+import com.google.common.collect.Interners
+import com.wavesplatform.common.state.ByteStr
+import com.wavesplatform.common.utils.Base58
+import com.wavesplatform.crypto
+import com.wavesplatform.transaction.ValidationError.GenericError
+
+import scala.util.{Failure, Success}
+
+final case class KeyPair(seed: ByteStr) {
+  private[this] lazy val pair = crypto.createKeyPair(seed)
```

Comment:
```suggestion
lazy val (PrivateKey(privateKey), PublicKey(publicKey)) = crypto.createKeyPair(seed)
```
`unapply` needs to be tweaked tho